packages/jest-util/src/Console.js

```diff
@@ -48,8 +49,10 @@ export default class CustomConsole extends Console {
   }
 
   assert(...args: Array<any>) {
-    if (args[0]) {
-      this._log('assert', format(...args.slice(1)));
+    try {
+      assert(!!args[0], ...args.slice(1));
```

Comment:
let's just use `assert(...args)` since extra arguments are irrelevant to it. Same for buffered console.

Reply:
I agree, fixed